shared/actions/teams.tsx

```diff
@@ -529,42 +533,46 @@ function* getDetails(_, action: TeamsGen.GetDetailsPayload, logger) {
       return reqMap
     }, {})
 
-    const invites = map(details.annotatedActiveInvites, (invite: RPCTypes.AnnotatedTeamInvite) => {
-      const role = Constants.teamRoleByEnum[invite.role]
-      if (role === 'none') {
-        return null
-      }
-      const username = (() => {
+    const invites = Object.values(details.annotatedActiveInvites).reduce<Array<Types.InviteInfo>>(
+      (arr, invite) => {
+        const role = Constants.teamRoleByEnum[invite.role]
+        if (role === 'none') {
+          return arr
+        }
+
+        let username = ''
         const t = invite.type
-        if (t.c !== RPCTypes.TeamInviteCategory.sbs) {
-          return ''
+        if (t.c === RPCTypes.TeamInviteCategory.sbs) {
+          const sbs: RPCTypes.TeamInviteSocialNetwork = t.sbs || ''
+          username = `${invite.name}@${sbs}`
         }
-        const sbs: RPCTypes.TeamInviteSocialNetwork = t.sbs || ''
-        return `${invite.name}@${sbs}`
-      })()
-      return Constants.makeInviteInfo({
-        // @ts-ignore possibly a real issue
-        email: invite.type.c === RPCTypes.TeamInviteCategory.email ? invite.name : '',
-        id: invite.id,
-        // @ts-ignore possibly a real issue
-        name: invite.type.c === RPCTypes.TeamInviteCategory.seitan ? invite.name : '',
-        role,
-        username,
-      })
-    }).filter(Boolean)
+        arr.push(
+          Constants.makeInviteInfo({
+            // @ts-ignore TODO this may be a real problem
```

Comment:
figured out the issue, our code gen makes the wrong type. ticketed to work on this next